hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/OzoneContainer.java

```diff
@@ -183,8 +183,12 @@ private void stopContainerScrub() {
   public void start(String scmId) throws IOException {
     LOG.info("Attempting to start container services.");
     startContainerScrub();
```

Comment:
Do we need to do somthing similar for startContainerScrub also?

Reply:
Added.